uart-msgpacker/src/main/java/nl/isaac/androidlibs/uart_msgpacker/model/Responses.kt

```diff
@@ -4,14 +4,27 @@ import com.google.gson.annotations.SerializedName
 import java.io.Serializable
 
 enum class TimestampType(val id: String) {
-    OCCURRED("occured"), RESET("reset"), ROOTCAUSEDISAPPEARED("rootcausedisappeared"), END_OF_LIST("endoflist")
+    OCCURRED("occured"), RESET("reset"), ROOTCAUSEDISAPPEARED("rootcausedisappeared"), END_OF_LIST("endoflist"),
+    OCCURRED_NEW("occ"), RESET_NEW("rst"), ROOTCAUSEDISAPPEARED_NEW("dis"), END_OF_LIST_NEW("eol");
```

Comment:
I think TimestampType doesn't need the id property. It doesn't seem to be used in the code, just the enum itself.

Reply:
You are right, at the beginning I removed it completely, but who knows maybe you implemented this so in the future it might be used?

Reply:
I removed it